JumpscaleCore/servers/openresty/web_resources/static/weblibs

```diff
@@ -1 +1 @@
-/sandbox/code/github/threefoldtech/jumpscaleX_weblibs/static
\ No newline at end of file
+C:/sandbox/code/github/threefoldtech/jumpscaleX_weblibs/static
```

Comment:
this change seems wrong

Reply:
it's generated by windows, sorry for that, will remove